chromium_src/ios/chrome/browser/policy/model/configuration_policy_handler_list_factory.mm

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "ios/chrome/browser/policy/model/configuration_policy_handler_list_factory.h"
+
+#include <memory>
+
+#include "brave/ios/browser/policy/brave_simple_policy_map_ios.h"
+#include "components/policy/core/browser/configuration_policy_handler.h"
+
+#define BuildPolicyHandlerList BuildPolicyHandlerList_ChromiumImpl
+#include <ios/chrome/browser/policy/model/configuration_policy_handler_list_factory.mm>
```

Comment:
`#include "src/ios/chrome/browser/policy/model/configuration_policy_handler_list_factory.mm"`

Reply:
This is valid now for chromium_src overrides: https://github.com/brave/brave-core/pull/29652